guidelines/index.html

```diff
@@ -531,6 +531,34 @@ <h2>Statement of Partial Conformance - Language</h2>
                 <p>A "statement of partial conformance due to language" may be made when the page does not conform, but would conform if <a>accessibility support</a> existed for (all of) the language(s) used on the page. The form of that statement would be, "This page does not conform, but would conform to WCAG 2.2 at level X if accessibility support existed for the following language(s):"</p>
             </section>
 
+            <section class="informative" id="privacy-summary">
+                <h2>Privacy Considerations</h2>
+                <p>Success Criteria within this specification that may relate to privacy are:</p>
+                <ul>
+                    <li><a href="#timeouts">2.2.6 Timeouts (AAA)</a></li>
+                    <li><a href="#redundant-entry">3.3.7 Redundant Entry (A)</a></li>
+                </ul>
+
+            </section>
+
+            <section class="informative" id="security-summary">
+                <h2>Security Considerations</h2>
+                <p>Success Criteria within this specification that may relate to security are:</p>
+                <ul>
+                    <li><a href="#non-text-content">1.1.1 Non-text Content (A)</a></li>
```

Comment:
maybe worth adding to this, and 1.4.7, that these are `</a> (in the context of CAPTCHA challenges)</li>` or similar, just to clarify why they're included since it's non-obvious?